docs/api/browser-window.md

```diff
@@ -371,6 +371,18 @@ It creates a new `BrowserWindow` with native properties as set by the `options`.
       English and not localized.
     * `navigateOnDragDrop` Boolean (optional) - Whether dragging and dropping a
       file or link onto the page causes a navigation. Default is `false`.
+    * `autoplayPolicy` String (optional) - Autoplay policy to apply to
```

Comment:
can you make it an enum?
such as `options.mode` in `contents.openDevTools`?
```
  * `mode` String - Opens the devtools with specified dock state, can be
    `right`, `bottom`, `undocked`, `detach`. Defaults to last used dock state.
    In `undocked` mode it's possible to dock back. In `detach` mode it's not.
```

Reply:
Your current syntax should work @nornagon but would lose info about each value in the definitions anyway.  

I'll look into why this isn't being picked up as an enum